packages/core/drizzle/meta/_journal.json

```diff
@@ -1 +1,601 @@
\ No newline at end of file
+{
```

Comment:
Something weird happened in the previous PR

Reply:
prettier formatting jsons, nothing to worry about